packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -903,34 +903,54 @@ AccountSwitcher.propTypes = {
     account_list: PropTypes.array,
     account_loginid: PropTypes.string,
     accounts: PropTypes.object,
+    account_settings: PropTypes.object,
     account_type: PropTypes.string,
     can_change_fiat_currency: PropTypes.bool,
     can_upgrade_to: PropTypes.string,
+    client_residence: PropTypes.string,
     country_standpoint: PropTypes.object,
-    has_fiat: PropTypes.bool,
+    dxtrade_accounts_list: PropTypes.array,
+    dxtrade_accounts_list_error: PropTypes.string, // is this correct?
```

Comment:
<img width="877" alt="dxtrade_accounts_list_error" src="https://user-images.githubusercontent.com/104334373/177234758-a42ae67d-7d26-467e-8a13-a0070766b31c.png">

Shayan,

Based on it's usage in the snippet above, it seems like a boolean, but I guess it could be a string, since string values could be truthy as well. not sure about it. I think further investigation needs to be done.,

Reply:
@jim-deriv actually I wasn't quite sure about it. I will investigate it

Reply:
Yes Shayan,

I also have a few of them. If I turn up something, I'll let you know as I'm sure you'll let me know as well in case you do

Reply:
@jim-deriv I noticed that there is !! before dxtrade_accounts_list_error in this file. Ako also told me to use it here just to make sure there won't be any problems.

Reply:
Hello @iman-fs,

Ok. I have been observing most places in code where error is used and I've been seeing them as string for example, the snippet in the screenshots below. So maybe you were right. It could be a string here as well

<img width="901" alt="error 2" src="https://user-images.githubusercontent.com/104334373/178624099-f59b6f26-3558-401f-bb80-e5fb86071de8.png">


<img width="835" alt="error" src="https://user-images.githubusercontent.com/104334373/178624029-79c3ab78-40ef-428f-b865-7e7c72adccd0.png">